src/feoperator/projection_newapi.jl

```diff
@@ -73,10 +73,12 @@ get_fespace(cache::CellMeanCache) = cache.feSpace
 get_measure(cache::CellMeanCache) = cache.measure
 get_mass_matrix(cache::CellMeanCache) = cache.massMatrix
 
-function build_cell_mean_cache(u::AbstractSingleFieldFEFunction, dΩ::Measure)
+function build_cell_mean_cache(u::AbstractLazy, dΩ::Measure)
     mesh = get_mesh(get_domain(dΩ))
     fs = FunctionSpace(:Lagrange, 0)
-    ncomp = get_ncomponents(get_fespace(u))
+    _, N = get_return_type_and_codim(u, mesh)
+    @assert length(N) == 1 "ncomponents > 1 not supported yet"
```

Comment:
Does [it](https://github.com/bcube-project/BcubeTutorials.jl/blob/e5ef3e24c578e5cc3fb277cf08cfac89d057055d/src/example/euler_naca_steady/euler_naca_steady.jl#L239) still work with this limitation?

Reply:
It does. It's working for vectors (only one "axis") but will fail for tensors.